content/sensu-core/2.0/reference/filters.md

```diff
@@ -20,10 +20,10 @@ filter analysis flow performs these steps:
 
 * When the Sensu server is processing an event, it will check for the definition
 of a `handler` (or `handlers`). Prior to executing each handler, the Sensu
-server will first apply any configured `filter` (or `filters`) for the handler.
+server will first apply any configured `filters` for the handler.
 * If multiple `filters` are configured for a handler, they are executed
 sequentially.
-* Filter `statements` are compared with event data.
```

Comment:
I don't think this line should change, it's describing individual named filters.